Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -210,6 +212,49 @@ public void ProjectWideActions_CanReplaceExistingActions()
         Assert.That(InputSystem.actions.actionMaps[1].actions.Count, Is.EqualTo(1));
         Assert.That(InputSystem.actions.actionMaps[1].actions[0].name, Is.EqualTo("replacedAction4"));
     }
+
+    [Test]
+    [Category(TestCategory)]
```

Comment:
Great to get some testing on this in, we can likely extend this with Undo/Redo functionality later, e.g. https://docs.unity3d.com/ScriptReference/Undo.PerformUndo.html and https://docs.unity3d.com/ScriptReference/Undo.PerformRedo.html